File: 0389-find-the-difference/0389-find-the-difference.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        s = list(s)
        t = list(t)
        s.sort()
        t.sort()
        if len(s) > 1:
            for i in range(len(t)-1):
                if s[i] != t[i]:
                    return t[i]
            return t[-1]
        return t[0]
            
        

File: 0389-find-the-difference/0389-find-the-difference.js
/**
 * @param {string} s
 * @param {string} t
 * @return {character}
 */
var findTheDifference = function(s, t) {
    sArray = s.split('')
    tArray = t.split('')
    sArray.sort()
    tArray.sort()
    for(let i = 0; i < t.length;i++){
        if (sArray[i] !== tArray[i]) return tArray[i]
    }
    return tArray[ t.length-1]
    
    
};